frappe/installer.py

```diff
@@ -707,20 +682,6 @@ def convert_archive_content(sql_file_path):
 		old_sql_file_path.unlink()
 
 
-def extract_sql_gzip(sql_gz_path):
```

Comment:
Just check if it's not used anywhere. Sourcegraph is a nice tool for this. 

https://sourcegraph.com/search?q=context:global+extract_sql_gzip+repo:%5Egithub%5C.com/frappe/.*&patternType=standard&sm=0&groupBy=repo

Reply:
Ah cool, thanks.